2017/Dockerfile

```diff
@@ -36,12 +36,13 @@ RUN chmod 755 /usr/local/bin/start.sh
 
 # Setup for reading Health Insurance ID Card
 
-# Install script for plugin of Health Insurance ID Card will use killall command
-COPY health_insurance_card /tmp/health_insurance_card/
+ADD health_insurance_card/mLNHIICC_Setup.Ubuntu16.tar.gz /tmp/health_insurance_card/
```

Comment:
Is there a download link for mLNHIICC_Setup.Ubuntu16.tar.gz? I don't feel like redistribute the binary blobs.